tools/proofers/report_bad_page.php

```diff
@@ -31,17 +31,28 @@
 
     echo "<h1>$header</h1>";
 
-    echo "<h2>"._("Common Fixes for Bad Pages. Try these first!")."</h2>";
+    echo "<p>" . _("If you are unable to proofread the page you were presented, you can mark it bad with this form to let the Project Manager know it requires attention. Before doing so, let's review what constitutes a bad page and some possible fixes you can try first.") . "</p>";
+
+    echo "<h2>" . _("Commonly Misidentified Bad Pages") . "</h2>";
+    echo "<p>" . _("The following scenarios are commonly reported as bad pages, but they are not. If either of the below are true, return and continue proofreading the page.") . "</p>";
```

Comment:
Maybe "If either of the below are true, hit 'Cancel' below and continue proofreading the page.

Reply:
That sounds like a good idea.

---

Not something for this PR, but @70ray's comment reminded me of [Task #1972: Cancel Bad Page report fails if page was saved as in progress](https://www.pgdp.net/c/tasks.php?action=show&task_id=1972). I just verified that it's still happening. This will happen for anything that causes a save in progress (though I seem to be getting different error messages, depending on whether I cancel report bad page after report bad page after WC, or after changing from vertical to horizontal mode).

I don't know how important it is to get fixed -- I discovered it by accident, and nobody else seems to have stumbled over it, because generally they mark them bad before trying to proofread.